benchmarks/bench_isolation_forest.py

```diff
@@ -1,159 +0,0 @@
-"""
```

Comment:
I don't think this file should be removed. This benchmark is done in the novelty context which is not the case of the example

Reply:
Got it